pkg/timestamp/nano.go

```diff
@@ -69,6 +72,7 @@ var (
 
 	ErrTimeOutOfRange     = errors.Errorf("time is out of range %d - %d", MinNanoTime, MaxNanoTime)
 	ErrTimeNotMillisecond = errors.Errorf("time is not millisecond precision")
+	ErrTimeEmpty          = errors.Errorf("time is empty")
```

Comment:
These errors should contain gRPC status codes. Currently client cannot recognize these error types.

Reply:
done